runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -390,6 +392,35 @@ public void onStatChanged(StatChanged statChanged)
 		xpPanel.updateTotal(xpState.getTotalSnapshot());
 	}
 
+	private int gamemodeModifier()
+	{
+		final XpWorldType worldType = worldSetToType(client.getWorldType());
+
+		if (worldType == LEAGUE)
+		{
+			return (int) (worldType.getXpModifier() * relicModifier());
+		}
+
+		return worldType.getXpModifier();
+	}
+
+	private float relicModifier()
```

Comment:
I'd probably prefer to just see a `modifier()` overridden in the enum or something